.env

```diff
@@ -0,0 +1,12 @@
+# Should be loaded automatically by mise/asdf/direnv. This file does not
+# support the full shell syntax.
+#
+# PWD is supposed to be the directory container the .env file based on the
+# specs. This trick should isolate our testing from user environment.
+ANSIBLE_HOME=$PWD/.ansible
+ANSIBLE_CONFIG=$PWD/ansible.cfg
+
+# pytest-plus checks that we do not pass yet
+PYTEST_CHECK_TEST_DUPLICATE=0
```

Comment:
These would probably be better in the pyproject.toml along side the other configuration for pytest.

Reply:
Not possible right now as the plugin does only read env vars.